GitUtils.ps1

```diff
@@ -146,60 +148,87 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
-            dbg 'Getting status' $sw
-            $status = Invoke-Utf8ConsoleCommand { git -c color.status=false status --short --branch 2>$null }
-            if($settings.EnableStashStatus) {
-                dbg 'Getting stash count' $sw
-                $stashCount = $null | git stash list 2>$null | measure-object | Select-Object -expand Count
+            if ($settings.EnableFileStatusFromCache -eq $null) {
+                $settings.EnableFileStatusFromCache = (Get-Module GitStatusCachePoshClient) -ne $null

```

Comment:
It auto-selects based on the module's presence the first time through (when set to the default null value), but if you manually set it to false that'll stick on subsequent calls since it's no longer null.